docs/spec/conjure_definitions.md

```diff
@@ -355,7 +355,7 @@ An object representing an argument to an endpoint.
 
 Field | Type | Description
 ---|:---:|---
-type | [ConjureType][] | **REQUIRED**. The type of the value of the argument. The type name MUST exist within the Conjure definition. If this ArgumentDefinition has a param-type of `body` then there are no restrictions on the type. If the param-type is `path`, `header` or `query` then the de-aliased type MUST be an enum, a primitive or an optional of one of these. Additionally, the type MUST NOT be `bearertoken` when the param-type is `header` or `query`.
+type | [ConjureType][] | **REQUIRED**. The type of the value of the argument. The type name MUST exist within the Conjure definition. If this ArgumentDefinition has a param-type of `body` then there are no restrictions on the type. If the param-type is `path`, `header` or `query` then the de-aliased type MUST be an enum, a primitive or an optional of one of these. `query` type parameters may accept lists and sets in addition to optional containers, which are represented by repeating the `param-id`, for example `?key=value1&key=value2&key=value3`. Additionally, the type MUST NOT be `bearertoken` when the param-type is `header` or `query`.
```

Comment:
@cakofony I think I'd prefer to keep the actual examples in `wire.md` to minimize any distractions in this one.

Reply:
Makes sense, sorry!